docs/submit-transactions.md

```diff
@@ -111,8 +111,7 @@ can either be specified as options at the end of a command or as environment var
 
     Mandatory for all transactions. The author option lets you specify the account that is
     used to sign the transaction. You can use the `account` command group to manage your
-    accounts.
-    that you want to use to sign a transaction.
+    accounts, that you want to use to sign a transaction.
```

Comment:
This is unclear if it's referring to managing the accounts (i.e, if you want to sign a transaction, you need to manage your accounts), or to the accounts themselves. I suggest making this block:

 > Mandatory for all transactions. The author option lets you specify the local account that 
> is used to sign the transaction. You can use the `account` command group to generate 
> and list the local accounts that you can to use to sign transactions.

Reply:
I've just replaced the dot with the comma D: Your proposition is fine though :+1:

Reply:
Fixed